packages/block-editor/src/components/block-preview/index.js

```diff
@@ -1,10 +1,15 @@
 /**
  * WordPress dependencies
  */
+/**
+ * External dependencies
+ */
+import React from 'react';
```

Comment:
Why importing `react`? `createRef` is available in `@wordpress/element`

Reply:
Code review ftw. Thanks 👍

Reply:
Fixed